meson/post_install.py

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env python3
+import os, subprocess
+
+install_prefix = os.environ['MESON_INSTALL_PREFIX']
+schemadir = os.path.join(install_prefix, 'share/glib-2.0/schemas')
+
+if not os.environ.get('DESTDIR'):
+  print('Compiling GSettings Schemas ... ')
```

Comment:
```suggestion
  print('Compiling GSettings Schemas…')
```

We can use ellipses instead of three dots.
(For more details: https://elementary.io/docs/human-interface-guidelines#using-ellipses)